examples/CMakeLists.txt

```
@@ -22,6 +22,16 @@ endif()
 append(CMAKE_C_FLAGS "${CMAKE_EXAMPLE_CCXX_FLAGS}")
 append(CMAKE_CXX_FLAGS "${CMAKE_EXAMPLE_CCXX_FLAGS}")
 
+# link examples directly to OCL
+if(DNNL_GPU_RUNTIME STREQUAL "OCL")
+    find_package(OpenCL REQUIRED)
+    if(OpenCL_LIBRARY)
+        list(APPEND EXTRA_SHARED_LIBS ${OpenCL_LIBRARY})
+    else()
+        message(FATAL_ERROR "OpenCL runtime is not found in the environment.")
```

Consider adding a bit more details to make the error message actionable.

```suggestion
        message(FATAL_ERROR "OpenCL SDK is not found. You can use OPENCLROOT CMake option to specify the path or disable building tests with ONEDNN_BUILD_EXAMPLES=OFF and ONEDNN_BUILD_TESTS=OFF."
```